ui/DesignSystem/Component/index.js

```diff
@@ -18,6 +18,7 @@ import Rad from "./Rad.svelte";
 import RadicleLogo from "./RadicleLogo.svelte";
 import RadioOption from "./RadioOption.svelte";
 import Remote from "./Remote.svelte";
+import SearchModal from "./SearchModal.svelte";
```

Comment:
As mentioned by @juliendonck and commented above, this should be lifted to global App infra and be treated like hotkeys and transaction center.